elasticsearch/probe/readiness.sh

```diff
@@ -20,7 +20,7 @@
 ES_REST_BASEURL=https://localhost:9200
 EXPECTED_RESPONSE_CODE=200
 secret_dir=/etc/elasticsearch/secret
-max_time=${max_time:-1}
+max_time=${max_time:-0.8}
```

Comment:
A too short timeout might cause more trouble than a longer timeout. In a cluster under load / slow traffic consecutive readiness probes could fail and incorrectly take down Elasticsearch when it is otherwise functional.